content/en/docs/concepts/configuration/configmap.md

```diff
@@ -224,9 +224,7 @@ data has the following advantages:
 - improves performance of your cluster by significantly reducing load on kube-apiserver, by
 closing watches for config maps marked as immutable.
 
-To use this feature, enable the `ImmutableEphemeralVolumes`
```

Comment:
How about 

```
This feature is controlled by the `ImmutableEphemeralVolumes`
[feature gate](/docs/reference/command-line-tools-reference/feature-gates/),
which is enabled by default since v1.19. You can create an immutable
ConfigMap by setting the `immutable` field to `true`. For example,
```

I'm proposing a different saying because the gate is still there, although defaults to enabled. However, users can still turn it off if they want.

Reply:
Sounds good, fixed it as suggested :+1: